<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesReportingDialog</class>
 <widget class="QDialog" name="SalesReportingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sales Reporting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="monthPushButton">
     <property name="text">
      <string>Month</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="dayPushButton">
     <property name="text">
      <string>Day</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="yearPushButton">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>198</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="nextPushButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="prevPushButton">
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
